### PR TITLE
Eval tracker

### PR DESCRIPTION
**Pull request by kangben258**
# Scripts are provided for evaling the tracker.
## A note on scripts
At the end of the test, two metrics will be obtained, success(0-1)and precision(0-1), success represents the tracker's score of successfully tracking the target, the larger the value of success means the tracker is more robust and better. precision represents the tracker's accuracy score, the larger the value of precision means the tracker is more accurate.
## Here are the test results for DaSiamRPN

|       model      |    success     |    precision    |
| ---------------|--------------|----------------|
|  DaSiamRPN  |      0.322      |       0.409       |

### CODE REVIEW

**Review by zihaomu — COMMENTED** [review]
Please clean the meanfulless comment code.

**Comment by kangben258** [comment]
Scripts are provided for evaling the tracker.
Refer to tools/eval/README.md for usage.
At the end of the test, two metrics will be obtained, success(0-1) and precision(0-1), success represents the tracker's score of successfully tracking the target, the larger the value of success means the tracker is more robust and better. precision represents the tracker's accuracy score, the larger the value of precision means the tracker is more accurate. 
The scores of DaSiamRPN are 0.322 for success and 0.409 for precision.

File: tools/eval/README.md [hunk 1]
```diff
@@ -211,4 +212,32 @@ Run evaluation on quantized model with the following command :
...
 python eval.py -m pphumanseg_q -d mini_supervisely -dr /path/to/pphumanseg
-```
\ No newline at end of file
+```
+
+## OTB
+
+### Prepare data
+
+Please visit [here](https://drive.google.com/drive/folders/1DZvtlnG9U94cgLD6Yi3eU7r6QZJkjdl-?usp=sharing) to download the OTB dataset and the json file. Organize files as follow:
```
Comment (fengyuentau) [inline tools/eval/README.md:221]: Is this link provided by otb official or your google drive?

**Comment by fengyuentau** [comment]
> At the end of the test, two metrics will be obtained, success(0-1) and precision(0-1), success represents the tracker's score of successfully tracking the target, the larger the value of success means the tracker is more robust and better. precision represents the tracker's accuracy score, the larger the value of precision means the tracker is more accurate.

You can place this along with dataset information in `tools/eval/readme.md`.

> The scores of DaSiamRPN are 0.322 for success and 0.409 for precision.

Please also add this in the model information in `models/object_tracking_dasiamrpn/readme.md`.

**Comment by ryan1288** [comment]
@fengyuentau I'm thinkin of taking this incomplete evaluation to the finish line. If that sounds good, I'll understand the current change before proposing work to finish the PR.

**Comment by fengyuentau** [comment]
> @fengyuentau I'm thinking of taking this incomplete evaluation to the finish line. If that sounds good, I'll understand the current change before proposing work to finish the PR.

Sure, feel free to take this.

**Pull request closed without merging**